### ISSUE TITLE
Enhancement request: Email Meta Data CC only (in Addressee)

### ISSUE THREAD

**Issue by stf675** [osTicket/osTicket#2204]
Hi, currently we are able to make a ticket rule by choosing Email Meta Data Addressee (To or CC).

It will be extremely useful if Email Meta Data Addressee (CC Only) be available.  

It would allow to prevent duplicate ticket creation.  when  "ticket source" mailbox is CC and ticket is not meant to be created.

Thanks

**Comment by mckaygerhard** [comment]
i currently have similar problems in osticket 1.9.X, if user send email TO mail1 and added a CC mail2 and both mails are part of a sistem mail of differents departaments.. two tickets are created...

in part this its good, due maybe theres a problem of those two agents/departaments.. 

so a right solution its a flag on the system to allow duplication of differents departaments or duplication on same departaments.. (two flags in right)

**Comment by greezybacon** [comment]
I don't know what's up with your systems. The software is specifically designed not to create multiple tickets from a single email, no matter which headers are used to facilitate the email arriving in the system

**Comment by harrisyn** [comment]
@greezybacon I posted a separate issue relates to this.  Lots of duplication for me

**Comment by stf675** [comment]
let me clarify the issue we have. (to prevent subject hijack)

before we start using our "support" email address with osticket to create tickets, it was used to forward email to whole support desk.

it is also used to CC support desk as FYI.

when it happened, we do not want a ticket to be created.  currently I see no way to block ticket creation, when message arrive to support mailbox via CC  because current rule is "TO or CC"

If it is easy enough to be able to filter by CC only, it will add much more flexibility to ticket filters rules.

**Comment by mckaygerhard** [comment]
well @stf675 i have same problem, i have two departaments (two emails, support and systems) and each departaments have own email to create tickets.. 

if a user send a request to both (using cc or not) two tickets are created!

its a issue confirmed.

**Comment by mckaygerhard** [comment]
@greezybacon a better form to explain: if i added a main to CC i't suppost to be that email in oxticket will be added as collaborator, not open a new ticket!

**Comment by greezybacon** [comment]
> when it happened, we do not want a ticket to be created. currently I see no way to block ticket creation, when message arrive to support mailbox via CC because current rule is "TO or CC"

Can you forward it to something like `support+pleaseignore@mycompany.com`, and then add a filter for that email address and reject the tickets. Most mail systems support the concept of additions to the mailbox (`+pleaseignore` for instance) and will deliver the mail to the `support@mycompany.com` mailbox.

**Comment by stf675** [comment]
but I do want to create a ticket from email when it TO, and only reject when it CC.

**Comment by ManZosh** [comment]
Is this a dead issue by now? I'm in the same situation where being able to create a filter that only creates tickets from our designated email account being in the To: field of an email would be immensely helpful.

**Comment by JediKev** [comment]
@ManZosh 

**Edit:**
Just realized that you want the two separated.

Cheers.

**Comment by ManZosh** [comment]
No... I thought this thread is clearly asking to break up that option into separate To and CC options? Not both of them together. I want to reject emails addressed to our help desk email in the CC field but NOT in the To field?

**Comment by JediKev** [comment]
Updated my comment ^
